Data/Graph.hs

```diff
@@ -23,53 +23,76 @@
 -- Maintainer  :  libraries@haskell.org
 -- Portability :  portable
 --
--- A version of the graph algorithms described in:
+-- = Finite Graphs and Strongly Connected Components
```

Comment:
Strongly connected components don't go in the title. They're strongly connected components *of graphs*.

Reply:
Removed.